### ISSUE TITLE
Missing .config on Ubuntu throws error

### ISSUE THREAD

**Issue by edsu** [DocNow/twarc#431]
There has been a report that when running as root on a new Ubuntu system, if the .config directory is missing twarc2 will throw an error?

See: https://twitter.com/rabindra_lamsal/status/1380762481212026882

**Comment by edsu** [comment]
Testing with an Ubuntu docker image.

    docker run -i -t ubuntu /bin/bash
    apt update
    apt upgrade
    apt install python3 python3-pip 
    pip3 install twarc
    twarc configure

Here is the stack trace after twarc2 tries to save the entered bearer token:

```
  File "/usr/local/bin/twarc2", line 8, in <module>
    sys.exit(twarc2())
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1256, in invoke
    Command.invoke(self, ctx)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/twarc/command2.py", line 96, in twarc2
    ctx.invoke(configure)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/twarc/command2.py", line 111, in configure
    config_dir.mkdir()
  File "/usr/lib/python3.8/pathlib.py", line 1284, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/root/.config/twarc'
```

**Comment by edsu** [comment]
I just released a fix for this as part of v2.0.5 on PyPI.
